File: acct-gui/src/app/model/monthly-balance-report-record.ts
export class MonthlyBalanceReportRecord {
    public recordYearMonth : number;
    public startingBalance : number;
    public incomingBalance : number;
    public outgoingBalance : number;
    public remainingBalance : number;

    constructor(recordYearMonth:number, startingBalance:number, incomingBalance:number, outgoingBalance:number) {
        this.recordYearMonth = recordYearMonth;
        this.startingBalance = startingBalance;
        this.incomingBalance = incomingBalance;
        this.outgoingBalance = outgoingBalance;
        this.remainingBalance = this.startingBalance + this.incomingBalance + this.outgoingBalance;
    }
}

File: acct/acct-service/src/main/java/com/desolatetimelines/acct/service/model/MonthlyBalanceReportRecord.java
package com.desolatetimelines.acct.service.model;

import java.io.Serializable;

public class MonthlyBalanceReportRecord implements Serializable {
	private int recordYearMonth;

	private Double startingBalance;

	private Double incomingBalance;

	private Double outgoingBalance;

	public int getRecordYearMonth() {
		return recordYearMonth;
	}

	public void setRecordYearMonth(int recordYearMonth) {
		this.recordYearMonth = recordYearMonth;
	}

	public Double getStartingBalance() {
		return startingBalance;
	}

	public void setStartingBalance(Double startingBalance) {
		this.startingBalance = startingBalance;
	}

	public Double getIncomingBalance() {
		return incomingBalance;
	}

	public void setIncomingBalance(Double incomingBalance) {
		this.incomingBalance = incomingBalance;
	}

	public Double getOutgoingBalance() {
		return outgoingBalance;
	}

	public void setOutgoingBalance(Double outgoingBalance) {
		this.outgoingBalance = outgoingBalance;
	}

}
